intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_types.h

```diff
@@ -78,6 +78,7 @@ enum model_archs {
   MODEL_BAICHUAN,
   MODEL_CHATGLM2,
   MODEL_CHATGLM,
+  MODEL_WHISPER
```

Comment:
Why add here? I think whisper has its model_arch which is `struct whisper_model::type`. The `model_archs` here are actually "text generation model archs".

Reply:
refer to 
intel_extension_for_transformers/llm/runtime/graph/application/audio_run.cpp    line 748 
we need get model arch and may use for pybinding

Reply:
refer to intel_extension_for_transformers/llm/runtime/graph/application/audio_run.cpp line 748

`params.model_name` cannot be "non-whisper" as the params here are of type `whisper_param`. How can a `whisper_param` have model_arch of non-whisper?

Reply:
As long as one is in `audio_run.cpp` (or maybe future `audio_pybind.cpp`), it will have no access of `model_context_params::arch` (of type `enum model_archs`) or `struct model_struct::arch`(of type `enum model_archs`) but only access of `whisper_context` (no arch name field) and `struct whisper_model` (of type `enum e_model`).

Whisper does not have its equivalent of `model_context_params` as it just `whisper_init_from_file(const char* path_model)`.

Reply:
Yes, you're right, I already remove these code.